client/src/blog/post.tsx

```diff
@@ -191,11 +193,18 @@ export function BlogPost(props: HydrationData) {
   return (
     <>
       {doc && blogMeta && (
-        <>
-          <article
-            className="blog-container post container main-page-content"
-            lang={doc?.locale}
-          >
+        <main className="blog-post-container container main-page-content">
+          <div className="sidebar-container">
+            <div className="toc-container">
+              <aside className="toc">
+                <nav>
+                  {doc.toc && !!doc.toc.length && <TOC toc={doc.toc} />}
+                </nav>
+              </aside>
```

Comment:
Do we want all of this to be conditional on `doc.toc && !!doc.toc.length`, or do this aside and nav need to be here permanently for the css to work with the placement properly?

Reply:
That's a copy of the article toc. And I'm already touching to many things here, sorry.